src/en/space-station-14/departments/security/proposals/reduced-metagaming.md

```diff
@@ -0,0 +1,234 @@
+  - When purchased/obtained, the user is given a box. Opening this box activates a UI similar to a Chameleon UI.
+  - Once a glove design is chosen, the user is given a pair of thieving gloves with that design.
+  - The gloves give off matching fibers. They are not Chameleon.
+- Non-engineering insulated gloves/budget insulated gloves now come in 3 different colors with corresponding colored fibers. They are still called "Insulated Gloves"/"Budget Insulated Gloves". 
+- Rule change: Security has knowledge of Thieving Gloves and may confiscate the gloves of anyone who has stolen or attempted to steal contraband.
+- Rule change: An additional rule is added: Security may *not* test Thieving Gloves by putting them on and trying to steal an item off someone.
+
+This means that a thief is safe from having their gloves lost before committing a crime. Detectives can no longer use holofibers to confirm a Thief antag, but must instead rely on the fibers of the gloves around the crime scene. The addition of additional insuls colors means selecting yellow insulated gloves is no longer a safe option to blend in with all the other yellow gloves.
+
+## The Thief Kit Problem
+
+```admonish quote
+Seeing someone having an omega soap and assuming they therefore have a storage and DNA scramble implant
+```
+```admonish quote
+Seeing someone with a briefcase and concluding they are a thief
+```
+```admonish quote
+Knowing what thieves have in each kit
+```
+
+This issue is quite apparent and probably one of the easier ones to catch, but that doesn't mean it's any less annoying for Security who have to pretend that the Thief they caught don't have certain items. This is mostly only relevant to some sets; finding out someone has a Chameleon jumpsuit makes it only natural to further investigate and find the other Chameleon clothing.
+
+The kits that suffer the most from this issue are the following:
+- Chemistry Set
+  - Contains Omega Soap, Syringe, Ephedrine bottle. These could betray the Storage and DNA Scrambler Implants.
+- Syndie Set
+  - Contains Master At Arms hat and Interdyne Cigarettes. These could betray the Agent ID and Emag.
+- Sleeper Set
+  - Contains a Pyjama hat and Nitrous Oxide tank. These could betray the Hypopen. The jumpsuit isn't as big of an issue since it can be cut into cloth.
+- Communicator Set
+  - Contains a briefcase filled with a thief figurine, neck gater, tacticool jumpsuit and jensen coat. These could betray the Master Comms Key, CyberPen and Voicemask.
+
+### Proposed Solution
+
+Cull the unnecessary fluff items from the set and make them the same level of discoverability. If it's desirable to have the thief hide stuff, include that in the items being recieved; implants require the thief to hide the implanter for example. If the Communicator Set should require the thief to hide a fluff thief figurine for example, instead make the the Master Comms Key come in a box that needs to be hid. 
```

Comment:
I think there is, once again, a simpler solution here, make the supposedly stealthy items actually stealthy. This includes things like making briefcases good extra storage and making more roles spawn with them (Lawyer, librarian, and other "classy business" service roles especially should have them if they don't already)

Additionally, is it *really an issue* that these items can be in theory found? A number of these are standard station items one could find in, say, maint, and blend in perfectly there if you don't want them. Many of them already pass the sniff test as things you can see under normal circumstances (though the omega soap should probably get swapped out.)

Reply:
Some items I agree pass the sniff test, such as the Syndicate blanket or Interdyne cigarette pack. Seeing one of those does not *immediately* raise alarm. But a thief figurine is basically a blaring red light that there is a thief on the station, especially since that kit specifically has a bunch of fluff that largely gets tossed away in the same pile.

It's stupid that I can find a vodka bottle in the medbay backroom and immediately think "A doctor has a storage implant". It's just pointless. One can probably blame it as skill issue on the Thief's part, but unlike an implanter which is intrinsically tied to an item acting on a vodka bottle just feels cheap.

Reply:
> Some items I agree pass the sniff test, such as the Syndicate blanket or Interdyne cigarette pack. Seeing one of those does not _immediately_ raise alarm. But a thief figurine is basically a blaring red light that there is a thief on the station, especially since that kit specifically has a bunch of fluff that largely gets tossed away in the same pile.
> 
> It's stupid that I can find a vodka bottle in the medbay backroom and immediately think "A doctor has a storage implant". It's just pointless. One can probably blame it as skill issue on the Thief's part, but unlike an implanter which is intrinsically tied to an item acting on a vodka bottle just feels cheap.

Is the thief figurine not standard maint loot? (I'd consider that a bug)

Any items that aren't standard maint loot should either be added to that pool or removed from thief kits, that I can agree on.

Reply:
Thief figurines *technically* can spawn on the map. But they're only available in a specific spawner meant to be used in Libraries and competes with all the other figurines, so in practice they're exceedingly rare. 
https://github.com/space-wizards/space-station-14/blob/0450a416e22651270b5dc8f270ac5ca6780b7075/Resources/Prototypes/Entities/Markers/Spawners/Random/toy.yml#L136